docker/Dockerfile

```diff
@@ -94,16 +94,21 @@ ARG PYTHON_VER
 ################################ Stage: base (intermediate build target; basis for all other images herein)
 
 FROM python:${PYTHON_VER}-slim as base
+ARG PYTHON_VER
 
 ENV PYTHONUNBUFFERED=1 \
     NAUTOBOT_ROOT=/opt/nautobot \
     prometheus_multiproc_dir=/prom_cache
 
 # Install all OS package upgrades and dependencies needed to run Nautobot in production
-# hadolint ignore=DL3005,DL3008,DL3013
+# hadolint ignore=DL3005,DL3008,DL3009,DL3013,DL4006
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install --no-install-recommends -y git mime-support curl libxml2 libmariadb3 openssl && \
+    apt-get install --no-install-recommends -y curl
```

Comment:
Need curl to install Node and it makes sense to clean up only once (see removal later).